static/js/components/EmploymentTab.js

```diff
@@ -22,9 +28,16 @@ class EmploymentTab extends React.Component {
         <EmploymentForm {...this.props} showSwitch={true} validator={employmentValidation} />
         <ProfileProgressControls
           {...this.props}
-          nextBtnLabel="Next"
-          isLastTab={false}
-          validator={employmentValidation}
+          nextBtnLabel="I'm Done!"
+          isLastTab={true}
+          validator={
+            combineValidators(
+              personalValidation,
+              educationValidation,
+              employmentValidation,
+              privacyValidation
```

Comment:
do we still want the privacy validator here? what would the user see if there were an error?